src/DropdownMenu.tsx

```diff
@@ -102,6 +143,21 @@ const DropdownMenu: DropdownMenu = React.forwardRef(
     const prefix = useBootstrapPrefix(bsPrefix, 'dropdown-menu');
     const [popperRef, marginModifiers] = usePopperMarginModifiers();
 
+    const alignClasses: string[] = [];
+    if (align) {
+      if (typeof align === 'object') {
+        DEVICE_SIZES.forEach((brkPoint) => {
+          const direction = align[brkPoint];
+          if (direction) {
+            alignRight = alignRight || direction === 'left';
+            alignClasses.push(`${prefix}-${brkPoint}-${direction}`);
+          }
+        });
+      } else if (align === 'right') {
+        alignRight = true;
```

Comment:
**suggestion:** if we're deprecating `alignRight`, it might be better to implement `alignRight` in terms of setting `align = 'right'`

Reply:
Originally I was thinking when we do remove `alignRight` from the API, I would just use this var locally so I can toggle the `dropdown-menu-right` class in the case of responsive left alignment.  See comment above re: responsive left classes requiring this class.

Would this be ok?